rust/operator-binary/src/crd.rs

```diff
@@ -121,6 +129,37 @@ impl AutoTlsCa {
     }
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct CertManagerBackend {
+    /// A reference to the cert-manager issuer that the certificates should be requested from.
+    pub issuer: CertManagerIssuer,
+
+    /// The default lifetime of certificates.
+    ///
+    /// Defaults to 1 day. This may need to be increased for external issuers that impose rate limits (such as Let's Encrypt).
+    #[serde(default = "CertManagerBackend::default_certificate_lifetime")]
+    pub default_certificate_lifetime: Duration,
```

Comment:
We put this in `CertificateSpec.duration`, the docs on that state `Duration is the duration for the not after date for the requested certificate. this is set on order creation as pe the ACME spec.`.

Is this really a default value or the effective lifetime? In the later case I would suggest to rename it to `certificate_lifetime`

Reply:
On our end, it is the default value that can be overridden by the volume selector, just like for autoTls.

Reply:
Makes total sense, thanks!